src/main/java/com/spartronics4915/frc2020/commands/RaisePanelRotatorCommand.java

```diff
@@ -0,0 +1,39 @@
+package com.spartronics4915.frc2020.commands;
+
+import com.spartronics4915.frc2020.subsystems.PanelRotator;
+
+import edu.wpi.first.wpilibj.DriverStation;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+
+public class RaisePanelRotatorCommand extends CommandBase {
+
+    private PanelRotator mPanelRotator;
+    // You should only use one subsystem per command. If multiple are needed, use a CommandGroup.
+    public RaisePanelRotatorCommand(PanelRotator mSubsystem) {
+      mPanelRotator = mSubsystem;
```

Comment:
addRequirements

Reply:
>Inside of your constructor, you need to tell the CommandScheduler what subsystem your Command requires, so that it doesn't schedule two for the same subsystem. In this case, you'd call addRequirements(mPanelRotator).

Reply:
@j-james see my latest push. I think that does what you asked?

Reply:
Looks good.